sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -15216,2504 +15031,2006 @@
-              "showCharCount": false,
-              "showWordCount": false,
-              "allowMultipleMasks": false,
-              "addons": [],
-              "tree": false,
-              "lazyLoad": false,
-              "id": "e887kxl"
+              "width": 6,
+              "offset": 0,
+              "push": 0,
+              "pull": 0
+            }
+          ],
+          "type": "columns",
+          "hideLabel": true,
+          "tags": [],
+          "conditional": {
+            "show": "",
+            "when": null,
+            "eq": ""
+          },
+          "properties": {}
+        }
+      ],
+      "placeholder": "",
+      "prefix": "",
+      "suffix": "",
+      "multiple": false,
+      "defaultValue": null,
+      "protected": false,
+      "unique": false,
+      "persistent": false,
+      "clearOnHide": false,
+      "refreshOn": "",
+      "redrawOn": "",
+      "dataGridLabel": false,
+      "labelPosition": "top",
+      "description": "",
+      "errorLabel": "",
+      "autofocus": false,
+      "dbIndex": false,
+      "customDefaultValue": "",
+      "calculateValue": "",
+      "calculateServer": false,
+      "widget": null,
+      "validateOn": "change",
+      "validate": {
+        "required": false,
+        "custom": "",
+        "customPrivate": false,
+        "strictDateValidation": false,
+        "multiple": false,
+        "unique": false
+      },
+      "allowCalculateOverride": false,
+      "encrypted": false,
+      "showCharCount": false,
+      "showWordCount": false,
+      "allowMultipleMasks": false,
+      "addons": [],
+      "tree": false,
+      "lazyLoad": false,
+      "id": "ec2nq79"
+    },
+    {
+      "title": "Next steps",
+      "labelWidth": "",
+      "labelMargin": "",
+      "theme": "default",
+      "tooltip": "",
+      "customClass": " pt-3",
+      "collapsible": false,
+      "hidden": false,
+      "hideLabel": true,
+      "disabled": false,
+      "modalEdit": false,
+      "key": "ifYouveAlreadyHadALoanPanel",
+      "tags": [],
+      "properties": {},
+      "customConditional": "",
+      "conditional": {
+        "json": "",
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "logic": [],
+      "attributes": {},
+      "overlay": {
+        "style": "",
+        "page": "",
+        "left": "",
+        "top": "",
+        "width": "",
+        "height": ""
+      },
+      "type": "panel",
+      "label": "Panel",
+      "breadcrumb": "default",
+      "tabindex": "",
+      "input": false,
+      "tableView": false,
+      "components": [
+        {
+          "label": "Your Assessment",
+          "labelWidth": "",
+          "labelMargin": "",
+          "tag": "h2",
+          "className": "category-header-large primary-color pt-3",
+          "attrs": [
+            {
+              "attr": "",
+              "value": ""
             }
           ],
-          "collapsed": true,
+          "content": "Next steps",
```

Comment:
Can you please review the "Next steps" section. It seems different from Figma.
![image](https://github.com/user-attachments/assets/875a25ec-24d1-4270-867a-ad0b3c37434c)

Reply:
**accept** is not in bold as Figma shows.